projects/ngrx.io/content/examples/store-walkthrough/src/app/app.module.ts

```diff
@@ -2,20 +2,22 @@ import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
 
 // #docregion imports
-import { HttpClient, HttpClientModule } from '@angular/common/http';
+import { HttpClientModule } from '@angular/common/http';
 import { booksReducer } from './state/books.reducer';
 import { collectionReducer } from './state/collection.reducer';
 import { StoreModule } from '@ngrx/store';
 // #enddocregion imports
 
+// #docregion declareComponents
+import { CommonModule } from '@angular/common';
 import { AppComponent } from './app.component';
 import { BookListComponent } from './book-list/book-list.component';
 import { BookCollectionComponent } from './book-collection/book-collection.component';
 
-// #docregion declareComponents
 @NgModule({
   imports: [
     BrowserModule,
+    CommonModule,
```

Comment:
`CommonModule` should not be needed here, because `BrowserModule` already re-exports `CommonModule`: https://angular.io/guide/ngmodule-faq#should-i-import-browsermodule-or-commonmodule

https://github.com/ngrx/platform/issues/3542#issuecomment-1264735285

Reply:
ah.. you're right.. I was getting an error about how the async pipe could not be found when I tried the walkthrough the other day on stackblitz and it worked after adding the CommonModule but I don't see the error today.  I'll remove it as part of my updates.